README.md

```diff
@@ -43,11 +43,11 @@ To get started with the Swift Quickstart application follow these steps:
 
 <img width="700px" src="images/quickstart/xcode-video-quickstart.png"/>
 
-2. Type in an identity and click on "Generate Access Token" from the [Testing Tools page](https://www.twilio.com/console/video/runtime/testing-tools).
+2. Generate an Access Token
 
-<img width="700px" src="images/quickstart/generate_access_tokens.png"/>
+View instructions here for how to [generate an Access Token using the Twilio CLI](https://www.twilio.com/docs/video/tutorials/user-identity-access-tokens#generate-cli). Generating Access Tokens with the Twilio CLI is a quick way to get an Access Token for testing and development purposes.
```

Comment:
Access tokens can still be generated via the console. But the only way to access it is from the URL below. I don't think there is a way to navigate to it within console. Should we still list the console option? Should we try to have console add a navigation link?

https://console.twilio.com/us1/develop/video/manage/video-credentials?frameUrl=/console/video/project/testing-tools

Reply:
Oh interesting, I'd thought it had been removed.

It looks like ultimately it's supposed to be removed from the console, at least from what I've seen in Jira: https://issues.corp.twilio.com/browse/VNVCON-5294